src/compliment/sources/local_bindings.clj

```diff
@@ -68,13 +68,11 @@
         (not (#{'& '_} binding-node))
         (let [result binding-node
               candidate (delay (extract-tag-from-bound-to ns bound-to))]
```

Comment:
You left the delay here. Also, is there a reason to rename binding-node to result? Because it is shorter?